karma.conf.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(config) {
+  config.set({
+
+    basePath: '.',
+
+    frameworks: ['browserify', 'jasmine'],
+
+    browserify: {
+      debug: true,
+      transform: [
+        ['babelify', {presets: ['es2015']}]
+      ]
+    },
+
+    files: [
+      'tests/*-spec.js'
+    ],
+
+    preprocessors: {
+      'tests/*-spec.js': 'browserify'
+    },
+
+    reporters: ['progress', 'dots'],
+
+    port: 9876,
+
+    colors: true,
+
+    logLevel: config.LOG_INFO,
+
+    autoWatch: true,
```

Comment:
I prefer having watch be an option, so when running just `karma`, it'll go through the specs and end.